r_scripts/data_wrangle/2013_oregon_diagnosis_separate.R

```diff
@@ -0,0 +1,147 @@
+# ------------------------------------------------------------------------------
```

Comment:
Is part of this script duplicate from above script? Consider removing the above script if so.